drivers/bus/isapnp/hardware.c

```diff
@@ -791,21 +739,13 @@ ParseTags(
                 if (LogDev != 0 || DfState != dfStarted)
                     goto SkipTag;
 
+                Resource->Type = ISAPNP_RESOURCE_TYPE_END_DEPEDENT;
+                Resource++;
```

Comment:
Sometimes you use `++Resource;` and other times you use `Resource++;`, can you make your changes more homogeneous regarding that?

Reply:
Done.